src/Microsoft.IdentityModel.Tokens/CryptoProviderFactory.cs

```diff
@@ -111,11 +111,17 @@ public CryptoProviderFactory(CryptoProviderFactory other)
         /// </summary>
         /// <param name="key">the <see cref="SecurityKey"/> to use.</param>
         /// <param name="algorithm">the algorithm to use.</param>
+        /// <exception cref="ArgumentNullException">thrown if <paramref name="key"/> is null.</exception>
+        /// <exception cref="ArgumentNullException">thrown if <paramref name="algorithm"/> is null or empty.</exception>
+        /// <exception cref="ArgumentException">thrown if If <see cref="SecurityKey"/> and algorithm pair are not supported.</exception>
```

Comment:
nit: extra "if"

Reply:
thanks.

Reply:
https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/commit/dd4056bb650b8ec7a48d45c0e62e6454da1c7736